ImageClassification/ViewControllers/CameraBoard.swift

```diff
@@ -150,10 +150,13 @@ extension CameraBoard: CameraFeedManagerDelegate {
 	}
 	
 	func didOutput(pixelBuffer: CVPixelBuffer) {
-//        let currentTimeMs = Date().timeIntervalSince1970 * 1000
-//        guard (currentTimeMs - previousInferenceTimeMs) >= delayBetweenInferencesMs else { return }
-//        previousInferenceTimeMs = currentTimeMs
-
+		// This block of code set a delay between inferences.
+		/*
+        let currentTimeMs = Date().timeIntervalSince1970 * 1000
+        guard (currentTimeMs - previousInferenceTimeMs) >= delayBetweenInferencesMs else { return }
+        previousInferenceTimeMs = currentTimeMs
+		*/
+		
```

Comment:
@likhongomes Pay close attention to these lines because that is where there is a programmatic delay in inference.